<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="315897" started-at="2020-01-03T09:17:01Z" finished-at="2020-01-03T09:22:17Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="315897" started-at="2020-01-03T09:17:01Z" finished-at="2020-01-03T09:22:17Z">
      <class name="Paytm.MobileAutomation.Testcases.PaytmLoginTest">
        <test-method status="PASS" signature="AppiumserStart()[pri:0, instance:Paytm.MobileAutomation.Testcases.PaytmLoginTest@eec5a4a]" name="AppiumserStart" is-config="true" duration-ms="59479" started-at="2020-01-03T14:47:01Z" finished-at="2020-01-03T14:48:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AppiumserStart -->
        <test-method status="PASS" signature="Login()[pri:0, instance:Paytm.MobileAutomation.Testcases.PaytmLoginTest@eec5a4a]" name="Login" duration-ms="256386" started-at="2020-01-03T14:48:00Z" finished-at="2020-01-03T14:52:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="FAIL" signature="appiumserverStop()[pri:0, instance:Paytm.MobileAutomation.Testcases.PaytmLoginTest@eec5a4a]" name="appiumserverStop" is-config="true" duration-ms="0" started-at="2020-01-03T14:52:17Z" finished-at="2020-01-03T14:52:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Paytm.MobileAutomation.base.paytmBaseClass.appiumserverstop(paytmBaseClass.java:92)
	at Paytm.MobileAutomation.Testcases.PaytmLoginTest.appiumserverStop(PaytmLoginTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.afterRun(TestRunner.java:983)
	at org.testng.TestRunner.run(TestRunner.java:614)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumserverStop -->
      </class> <!-- Paytm.MobileAutomation.Testcases.PaytmLoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
